src/abi.h

```diff
@@ -269,6 +269,7 @@ struct netplan_net_definition {
         char* transmit_hash_policy;
         char* selection_logic;
         gboolean all_slaves_active;
+#define all_members_active all_slaves_active
```

Comment:
I feel like we should rather rename the struct field and make use of the new `all_members_active` variable name in our internal code. Same goes for `packets_per_member` below.

In any case, I'd make the new, inclusive name the default variable name and have the old, non-inclusive name be the `#define`/alias.

Reply:
I'll go with renaming them